modules/cli-variables.js

```diff
@@ -0,0 +1,17 @@
+const confFile = '/tmp/tezster/config.json',
+      jsonfile = require('jsonfile'),
```

Comment:
you can rename the file to cli-constants.js. 
also you just need to store constatant in the file. Exporting jsonfile is wrong with this design.
you can read config file with common helper function to avoid repeatation.

Reply:
please review the code, as I have made some changes regarding the same.